modules/@apostrophecms/ui/ui/apos/components/AposFile.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div>
+    <label
+      class="apos-input-wrapper apos-file-dropzone"
+      :class="{
+        'apos-file-dropzone--dragover': dragging,
+        'apos-is-disabled': disabled || limitReached
+      }"
+      @drop.prevent="uploadFile"
+      @dragover="dragHandler"
+      @dragleave="dragging = false"
+    >
+      <p class="apos-file-instructions">
+        <template v-if="dragging">
+          <cloud-upload-icon :size="38" />
+        </template>
+        <AposSpinner v-else-if="uploading" />
+        <template v-else>
+          <paperclip-icon :size="14" class="apos-file-icon" />
+          {{ messages.primary }}&nbsp;
+          <span class="apos-file-highlight" v-if="messages.highlighted">
+            {{ messages.highlighted }}
+          </span>
+        </template>
+      </p>
+      <input
```

Comment:
Yes I was too.. but I think you're tight, since there is no place where we use a multi files dropzone for now it makes no sense to allow it. + It wouldn't be that complicated to modify if needed..

Reply:
Modified to allow one selected file only.

Reply:
Agree this is OK for now.